### DISCUSSION TITLE
Is local pickup implemented?

### DISCUSSION THREAD

**Discussion by gleniat** [saleor/saleor-checkout#134] [General]
Hi,

is local pickup already implemented? If not yet, is it on the roadmap?

**Comment by timuric** [comment]
Saleor API supports local pickup, but we don't yet have plans for adding that to the checkout front end since most businesses have different pickup rules. Please feel free to share your use case so we could use it for future reference if we would decide to build this feature.

**Comment by gleniat** [comment]
Currently, my use case is the simplest one - a single branch store with a e-commerce website. The only one warehouse is the store itself.

With local pickup enabled on the warehouse, I expect it to show on checkout page as another delivery option like this:

<img width="725" alt="" src="https://user-images.githubusercontent.com/70897052/178059867-b869d239-2394-422a-8aaf-c61fe9072590.png">

**Comment by timuric** [comment]
@gleniat if it is just a matter of adding an option you could use shipping webhooks https://docs.saleor.io/docs/3.x/developer/extending/apps/synchronous-webhooks#shipping-webhooks
